### ISSUE TITLE
Allow through mzIdentML v1.2

### ISSUE THREAD

**Issue by Tobias-Ternent** [PRIDE-Archive/px-submission-tool#28]
Even though the mzIdentML v1.2 standard has not been finalised yet, the PEAKS software is producing 1.2 output (not 1.1). Currently only 1.1 is accepted with the submission tool, and through the PRIDE/PX data processing submission pipeline.

The current work-around is to rename 1.2 to 1.1 in the header section of each mzIdentML file to be submitted. The rest of the file is OK, because the 1.2 specification is only a little updated from 1.1, and the files generated from PEAKS are 1.1 compatible.
This is not ideal because a submitter might have to edit a lot of files manually, and may not have enough technical skills to write/run a script to do this in an automated way.
Also it causes some (unnecessary) frustration because the submission tool will stop the user with an error message, and that user may not know about mzIdentML version standards

To summarise:
PEAKS will not support exporting 1.1 directly, it will only support 1.2.
It would be a good to allow through mzIdentML 1.2 to make life a bit easier for submitters that use PEAKS.
Such files will still need to be manually edited to 1.1 by PRIDE/PX submission curators, since not already done so by submitters.
It is not a perfect solution.
